orte/mca/ess/pmi/ess_pmi_module.c

```diff
@@ -203,7 +203,7 @@ static int rte_init(void)
     /* get the number of local peers - required for wireup of
      * shared memory BTL */
     OPAL_MODEX_RECV_VALUE(ret, OPAL_PMIX_LOCAL_SIZE,
-                          ORTE_PROC_MY_NAME, &u32ptr, OPAL_UINT32);
+                          &name, &u32ptr, OPAL_UINT32);
```

Comment:
I meant the name of the variable. Name `name` is confusing to me. The following line would look much nicer for me:
```C
OPAL_MODEX_RECV_VALUE(ret, OPAL_PMIX_LOCAL_SIZE,
                          &wildcard, &u32ptr, OPAL_UINT32);
```